exercises/e001/results/Mesiya391/stringGenerator.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+import string
+import random
+
+parser = argparse.ArgumentParser()
+group = parser.add_mutually_exclusive_group(required=False)
+parser.add_argument("size", help="sets length of chain", nargs='?', const=32, type=int, default=32)
+group.add_argument("-l", "--lowercase",  help="set letters to lowercase",  action='store_true')
+group.add_argument("-u", "--uppercase", help = "set letters to uppercase", action='store_true')
```

Comment:
In the code you don't use `args.uppercase` because you used  `string.ascii_uppercase` and also `args.lowercase` and `args.uppercase` are mutually exclusive. This results that code works as expected in terms of lower / upper case characters.

However, to make code resistant to bugs in the future, in case of refactor, could you use for `-l` and `-u` the same destination variable and mark for on option`action='store_true'` and other option the opposite `action='store_false'`?

See `argparse` manual and `dest=`.